Resources/public/Admin.js

```diff
@@ -342,9 +342,9 @@ var Admin = {
             var collection = jQuery(this);
             var counter = 0;
             collection.children().each(function() {
-                var matches = highestCounterRegexp.exec(jQuery(this).find('.form-control').attr('id'));
+                var matches = highestCounterRegexp.exec(jQuery(this).find('[id^="sonata-ba-field-container"]').attr('id'));
```

Comment:
You can use `jQuery('[id^="sonata-ba-field-container"]', this)` to avoid instancing jQuery on `this`.